deploy/README.md

```diff
@@ -24,7 +24,8 @@ This playbook deploys GCS on a Kubernetes cluster. All of GCS, except for the St
 - etcd-cluster with 3 nodes
 - glusterd2-cluster with a pod on each kube node and configured to use the deployed etcd-cluster
 - glusterd2-client service providing a single rest access point to GD2
-- csi-driver (provisioner, nodeplugin, attacher) configured to use glusterd2-client service to reach GD2
+- glusterfs-csi-driver (provisioner, nodeplugin, attacher) configured to use glusterd2-client service to reach GD2
```

Comment:
I think we need to remove the driver components here because node plugin and attacher doesn't reach gd2 

only provision will contact gd2 directly

Reply:
Done